### PR TITLE
don't refresh row number each time. issue #1380

### PR DESCRIPTION
**Pull request by jackyq2015**

### CODE REVIEW

File: .settings/org.eclipse.wst.jsdt.core.prefs [hunk 1]
```diff
@@ -186,6 +186,7 @@ org.eclipse.wst.jsdt.core.formatter.insert_space_after_comma_in_superinterfaces=
 org.eclipse.wst.jsdt.core.formatter.insert_space_after_comma_in_type_arguments=insert
 org.eclipse.wst.jsdt.core.formatter.insert_space_after_comma_in_type_parameters=insert
 org.eclipse.wst.jsdt.core.formatter.insert_space_after_ellipsis=insert
+org.eclipse.wst.jsdt.core.formatter.insert_space_after_function_keyword=do not insert
```
Comment (wetneb) [inline .settings/org.eclipse.wst.jsdt.core.prefs]: Is that related to the bug?

File: main/src/com/google/refine/commands/row/GetRowsCommand.java [hunk 2]
```diff
@@ -180,7 +180,8 @@ protected void internalRespond(HttpServletRequest request, HttpServletResponse r
             }
             
             // metadata refresh for row mode and record mode
-	    if (project.getMetadata() != null) {
+	    if (project.getMetadata() != null
+	            && project.getMetadata().getRowCount() == 0) {
```
Comment (wetneb) [inline main/src/com/google/refine/commands/row/GetRowsCommand.java]: What if the row count changes later on (for instance if the user removes some rows, or uses a transpose operation)? Is the row count going to be updated somewhere else?

I would check if the row count is different from what is stored in the metadata, rather than comparing to zero.

Comment (jackyq2015) [reply main/src/com/google/refine/commands/row/GetRowsCommand.java]: The row number should be recounted after operation. Thanks for checking. just make sure.

File: .settings/org.eclipse.wst.jsdt.core.prefs (hunk 1 above)
Comment (jackyq2015) [reply .settings/org.eclipse.wst.jsdt.core.prefs]: no. for some reason, my eclipse automatically changed the setting while importing project

Comment (wetneb) [reply .settings/org.eclipse.wst.jsdt.core.prefs]: then I guess it's cleaner if we don't include that in the PR :)
